### ISSUE TITLE
Unread publications on screen reader

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#612]
On the “Notifications” screen, a red dot is used as the only visual means of conveying that there are new unread publications. Screen readers cannot navigate to the red dot and are not notified that there are unread publications. 

Programmatically ensure that screen reader users are made aware of which publications are unread.

![GitHub25](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/a86c4a8e-be1d-4fe9-9a66-d5cc9c760f3e)
